src/playbin.js

```diff
@@ -25,9 +26,22 @@ var Playbin = GObject.registerClass({
             Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION
         );
 
-        this.connect('button-release-event', () => {
+        /* Use a button to catch click event */
+        this._button = new Gtk.Button({visible: true});
+        this.add(this._button);
+
+        this._button.connect('clicked', () => {
             this.emit('clicked');
-            return Gdk.EVENT_PROPAGATE;
+        });
+
+        /* Make button invisible */
+        this._button.connect('draw', () => {
+            return true;
+        });
+
+        /* Update crop values on size allocation */
+        this.connect('size-allocate', (self, alloc) => {
+            self._updateCropArea(alloc);
```

Comment:
You can directly call `this._updateCropArea()`

Reply:
I don't think so, but if it does, you can simply add `void self;` or something like that

Reply:
if you use `alloc` then it will not complain about `self`

Reply:
Cool! You should be able to change this then